### PR TITLE
Add reading of CSV files to ModelicaStandardTables

### PR DESCRIPTION
**Pull request by beutlich**
Taken from prototype implementation of [ModelicaTableAdditions](https://github.com/modelica-3rdparty/ModelicaTableAdditions). Note, that no additional external 3-rd party dependencies are utilized. CSV tables with quoted values or line breaks are not supported.

### CODE REVIEW

**Comment by beutlich** [comment]
I update the binaries in my branch "update-binaries-add-csv". Use https://github.com/beutlich/ModelicaStandardLibrary/tree/update-binaries-add-csv if you want to test it.

**Comment by dietmarw** [comment]
Could it be that you forgot to add some test models that are reading the CSV files that were added?

**Comment by beutlich** [comment]
> 
> 
> Could it be that you forgot to add some test models that are reading the CSV files that were added?

Yes, I will add them, but wanted to gather some early feedback from library officers first.

**Comment by MartinOtter** [comment]
Sorry, currently have no time to inspect this in detail.

File: ModelicaTest/Resources/Data/Tables/test.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+time, value
+0.0, 0.0	
+1,1
```
Comment (HansOlsson) [inline ModelicaTest/Resources/Data/Tables/test.csv:3]: Why is there no end-of-line on the last line?
I see some problems with handling such files - and I certainly don't want to make that mandatory.

Comment (beutlich) [reply ModelicaTest/Resources/Data/Tables/test.csv:3]: This was taken from the original test file provided by @dietmarw. Both cases, that is with and w/o trailing new-line are covered. If you prefer, I can add it here, too.

**Comment by beutlich** [comment]
> 
> 
> I updated the binaries in my branch "update-binaries-add-csv". Use https://github.com/beutlich/ModelicaStandardLibrary/tree/update-binaries-add-csv if you want to test it.

@dietmarw I've just added the test models to ModelicaTest. If you want to test with Dymola 2021X make sure to set ``Advanced.PlaceDymolaSourceFirst = 2;` and use the provided binaries.

File: ModelicaTest/Resources/Data/Tables/test.csv (hunk 1 above)
Comment (beutlich) [reply ModelicaTest/Resources/Data/Tables/test.csv:3]: There are now also test models in ModelicaTest that refere to CSV files with trailing new-line. Please review again.

**Review by dietmarw — APPROVED** [review]
Approved. 
Note `Advanced.PlaceDymolaSourceFirst = 2;` was also necessary for Dymola 2020

**Review by HansOlsson — APPROVED** [review]
Seems good.
I tried rebuilding binaries and running test-case and the result matched the csv-file.

**Pull request merged**